src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    <p className="Person__age">
+      {person.age && (`I am ${person.age}`)}
+    </p>
+
+    <p className="Person__partner">
+      {person.sex === 'm' && person.isMarried === true && (
+        `${person.partnerName} is my wife`
+      )}
+
+      {person.sex === 'f' && person.isMarried === true && (
+        `${person.partnerName} is my husband`
+      )}
+
+      {person.isMarried === false && (
+        `I am not married`
+      )}
```

Comment:
Let's combine all these conditions using ternary operator. Create a variable for husband\wife and show the text with it or 'I am not married' message according to the isMarried boolean